extension/manifest.json

```diff
@@ -6,7 +6,9 @@
   "author": "Federico Brigante",
   "default_locale": "en",
   "permissions": [
-    "storage"
+    "storage",
+    "cookies",
```

Comment:
Why do you need to manually specify the cookies? They should already be sent with `credentials: 'include'`: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch